stock_barcodes/wizard/stock_barcodes_read_inventory.py

```diff
@@ -4,13 +4,20 @@
 from odoo.fields import first
 from odoo.addons import decimal_precision as dp
 from odoo.exceptions import ValidationError
+from odoo.tools.safe_eval import safe_eval
 
 
 class WizStockBarcodesReadInventory(models.TransientModel):
     _name = 'wiz.stock.barcodes.read.inventory'
     _inherit = 'wiz.stock.barcodes.read'
     _description = 'Wizard to read barcode on inventory'
 
+    def _default_auto_lot(self):
+        ICP = self.env['ir.config_parameter'].sudo()
+        auto_lot = ICP.get_param("stock_barcodes.auto_lot")
```

Comment:
Wouldn't it be better a company setting?

Reply:
Yes you are right, I have planned do it for future improvements.